apps/browser/src/vault/popup/components/fido2/fido2.component.ts

```diff
@@ -405,6 +406,16 @@ export class Fido2Component implements OnInit, OnDestroy {
     this.destroy$.complete();
   }
 
+  private async handleUserVerification(userVerification: boolean): Promise<boolean> {
+    const masterPasswordRempromptRequired = this.cipher && this.cipher.reprompt !== 0;
+
+    if (!masterPasswordRempromptRequired && !userVerification) {
+      return true;
+    }
+
+    return await this.passwordRepromptService.showPasswordPrompt();
+  }
+
```

Comment:
suggestion(non-blocking): Seems a bit odd to me that it take one part of data it needs as a parameter, and the other part it fetches straight from `this.`. I'd suggest choosing one of those approaches, preferably the functional approach by not using `this.` at all. 

```suggestion
  private async handleUserVerification(userVerification: boolean, cipher: CipherView): Promise<boolean> {
    const masterPasswordRempromptRequired = cipher && cipher.reprompt !== 0;

    if (!masterPasswordRempromptRequired && !userVerification) {
      return true;
    }

    return await this.passwordRepromptService.showPasswordPrompt();
  }

```